### ISSUE TITLE
Integrations for #accessibility

### ISSUE THREAD

**Issue by akirk** [Automattic/matrix-bots-wporg#11]
For the `#accessibility` Slack channel on Making WordPress, the following integrations need to be migrated from Slack to Matrix:

 - [ ] Workflow "Accessibility Welcome Bot"
 - [ ] Feed https://make.wordpress.org/accessibility/feed/
 - [ ] Trac Bot
